src/structures/MessageCollector.js

```diff
@@ -11,6 +11,7 @@ const { Events } = require('../util/Constants');
 
 /**
  * Collects messages on a channel.
+ * Will automatically stop if the Channel or Guild are deleted.
```

Comment:
Lowercase and name the reasons, as they are not publicly documented yet.
```suggestion
 * Will automatically stop if the channel (`'channelDelete'`) or guild (`'guildDelete'`) are deleted.
```